### ISSUE TITLE
{Linked:Bug8592964;Linked:Bug8592946} {Linked:Bug8592964}[Adaptive Cards Consumer->Dialog Header ACE UX]: The screen reader is not announcing the Dialog name "Vacation" at the transition end.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7201]
https://microsoft.visualstudio.com/OS/_workitems/edit/38787936

**Comment by anna-dingler** [comment]
@paulcam206 @licanhua Does it seem possible to you that this is our issue? The Dialog name that is not being read is not part of the AdaptiveCard the team is rendering

**Comment by anna-dingler** [comment]
Closing issue because the Dialog name "Vacation" does not belong to the AdaptiveCard. It should be handled be the host application. Thanks!
